api/transport/handler.go

```diff
@@ -96,6 +108,16 @@ type OnewayHandler interface {
 	HandleOneway(ctx context.Context, req *Request) error
 }
 
+// StreamHandler handles a stream connection request.
+type StreamHandler interface {
+	// Handle the given stream connection.  Optionally returns a Metadata field
+	// that includes the response headers and any other metadata passed to the
+	// client for the end of the connection.
+	//
+	// An error may be returned in case of failures.
+	HandleStream(stream ServerStream) error
```

Comment:
I presume you discarded InboundStream since it might be confused with the directions the stream carries messages, rather than which role this task is playing in initiating the stream.

Reply:
Actually, hadn't thought about it, this was the convention in grpc.  We sort of just pulled it over.